From ad0c6ec6c486882b5f442f9c5bf28d5d08588a58 Mon Sep 17 00:00:00 2001
From: kszaq <kszaquitto@gmail.com>
Date: Mon, 12 Sep 2016 17:00:00 +0200
Subject: [PATCH] Revert "[aocec] rename Class device to aocec for better
 detection"

This reverts commit 90cfad864188c3ba6fdbc2cb8fc2593fb3269550.
---
 drivers/amlogic/cec/hdmi_ao_cec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/cec/hdmi_ao_cec.c b/drivers/amlogic/cec/hdmi_ao_cec.c
index e3307cd..4c53100 100644
--- a/drivers/amlogic/cec/hdmi_ao_cec.c
+++ b/drivers/amlogic/cec/hdmi_ao_cec.c
@@ -66,7 +66,7 @@ static struct early_suspend aocec_suspend_handler;
 #include <linux/amlogic/jtag.h>
 
 #define CEC_FRAME_DELAY		msecs_to_jiffies(400)
-#define CEC_DEV_NAME		"aocec"
+#define CEC_DEV_NAME		"cec"
 
 #define DEV_TYPE_TV			0
 #define DEV_TYPE_RECORDER		1
